projects/allen-cell-segmenter.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Allen Cell Segmenter
+layout: page
+---
+
+- *Contributor(s)*: Jianxu Chen (@jxchen01), Benjamin Morris (@benjijamorris)
+- *References*: [Project Site](https://www.allencell.org/segmenter.html), [Github](https://github.com/AllenCell/aics-segmentation), [Publication](https://www.biorxiv.org/content/10.1101/491035v2), [Support](https://forum.allencell.org/c/software-code/segmenter/16)
+
+## Purpose
+
+> Describe the problem you are trying to solve by packaging your tool as a napari plugin in 1-2 sentences.
+I should be able to read this alone and communicate the value to someone else.
+
+### Why does this matter to biologists?
+
+### Why does this matter to you and your professional goals and responsibilities?
+
+## Goals
+> What would you like to accomplish during the Alfa Cohort collaboration?
+
+1. Make “lookup table” of segmentation algorithms available. Load workflow & tweak parameters based on instant feedback from the viewer. Target audience: Anyone
+2. [Stretch] For more advanced user who knows a bit about the algorithms, they have the freedom to manipulate the workflow. Freedom to use modules.
+3. [Ultra Stretch] Integrating with ML model. Curation of pass/fail (not painting). Use case: different areas of image may need different algorithm. Then a set of training data is ready for training.
+
+
+### Non-goals
+> Are there any possible goals that are explicitly out of scope for the Alfa Cohort?
+
+1. Not going to support 2D at this time
+
+## Scope
+
+### Key Flows
+
+> Show what the end-to-end experience will be for biologists.
+
+A biologist will be able to
+1. Load some 3D data
+2. Select a workflow which is tuned for a particular cell structure
```

Comment:
Check the workflow look-up tablle where each workflow is tuned for a particular structure. Pick the one showing the most similar morphology to user's data